modules/mauth-authenticator/src/test/scala/com/mdsol/mauth/RequestAuthenticatorSpec.scala

```diff
@@ -87,4 +89,13 @@ class RequestAuthenticatorSpec extends AnyFlatSpec with RequestAuthenticatorBase
     expectedException.getMessage shouldBe "The service requires mAuth v2 authentication headers."
   }
 
+  it should "validate a valid request with binary body for V1" in {
+    //noinspection ConvertibleToMethodValue
+    (mockEpochTimeProvider.inSeconds _: () => Long).expects().returns(CLIENT_X_MWS_TIME_HEADER_BINARY_VALUE.toLong + 3)
+    (mockClientPublicKeyProvider.getPublicKey _)
+      .expects(UUID.fromString(CLIENT_REQUEST_BINARY_APP_UUID))
+      .returns(MAuthKeysHelper.getPublicKeyFromString(PUBLIC_KEY2))
+    authenticator.authenticate(getRequestWithBinaryBodyV1) shouldBe true
+  }
+
```

Comment:
Probably a good idea to check that the string-body API and byte-array API will both succeed (they should be equivalent if their string body is the same)

Reply:
This is a test for the request with string payload
https://github.com/mdsol/mauth-jvm-clients/blob/384557bc88e9958b3295049bff5b310494ccd739/modules/mauth-authenticator/src/test/scala/com/mdsol/mauth/RequestAuthenticatorSpec.scala#L22

Reply:
Ah sorry I see you changed the implementation of validateSignatureV1 to always use bytes[] body